ArduCopter/mode.cpp

```diff
@@ -528,10 +529,11 @@ void Mode::land_run_vertical_control(bool pause_descent)
             cmb_rate = MIN(-precland_min_descent_speed, -max_descent_speed+land_slowdown);
         }
 #endif
+        force_descend = (MAX(g2.land_alt_low,100) > get_alt_above_ground_cm());
```

Comment:
just as a small style thing, I think we should move this above the, "#if PRECISION_LANDING == ENABLED" line.